recipes/dcmstack/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.6.2" %}
+
+package:
+  name: dcmstack
+  version: {{ version }}
+
+source:
+  fn: dcmstack-{{ version }}.tar.gz
+  url: https://github.com/moloney/dcmstack/archive/v{{ version }}.tar.gz
+  md5: 3409d0bcbb79e1790cb2639c7bfb1a6a
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - nibabel
+  run:
+    - python
+    - pydicom
+    - nibabel
+
+test:
+  imports:
+    - dcmstack
+  commands:
+    - python -c "import dcmstack;"
```

Comment:
No problem. It might be good to that in the example as well:
https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L20-L24